ginga/rv/plugins/Catalogs.py

```diff
@@ -8,41 +8,62 @@
 ``Catalogs`` is a local plugin, which means it is associated with a
 channel.  An instance can be opened for each channel.
 
+.. note:: to use ``Catalogs`` it is necessary to install the `astroquery`
+          package.
+
+.. note:: configuration of ``Catalogs`` via `ginga_config.py` technique
```

Comment:
I think this is important enough to be a warning.
```suggestion
.. warning:: Configuration of ``Catalogs`` via ``ginga_config.py`` technique in Ginga 3.2 or later
```